yugabytedb_managed/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "c5cf1ad4-fa3f-4835-9f3b-f467288b382a",
+  "app_id": "yugabytedb-managed",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Export YugabyteDB Managed cluster metrics to Datadog",
+    "title": "YugabyteDB Managed",
+    "media": [],
```

Comment:
Please include images for your integration tile. We highly recommend images of the graphs in the dashboard!

Reply:
Will work on getting this piece done first thing next week.

Reply:
Done.